src/pages/Touringcar/components/LocationsList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Fragment } from 'react'
+
+import { Table, TableCell, TableHeader, TableRow } from '@amsterdam/asc-ui'
+import { TouringcarStop } from 'api/touringcar/stops'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+const TableHeaderStrong = styled(TableHeader)`
+  font-weight: 700;
+`
+
+function LocationsList({ locationItems }: { locationItems: TouringcarStop[] }) {
+  const { t } = useTranslation()
+
+  return (
+    <Table>
+      <TableHeaderStrong>
+        <TableCell>{t('_pageTouringcar.reference')}</TableCell>
+        <TableCell>{t('_pageTouringcar.places')}</TableCell>
+      </TableHeaderStrong>
+
+      {locationItems.map(({ properties: item }, index) => {
+        if (!item) return <Fragment key={index}></Fragment>
+
+        return (
+          <TableRow key={item.omschrijving}>
```

Comment:
Are you sure `item.omschrijving` is always unique? Shouldn't you use the `index` here for the `key`?

Reply:
Well, it should be unique because it contains the stop/parking space number and name, i.e. "P3: Museumplein".